02-express-tutorial/answers/15-router-controller.js

```diff
@@ -0,0 +1,66 @@
+let { people } = require('../data')
```

Comment:
Since you have this file under `controllers/people.js` it doesn't need to be copied here as well.

Reply:
I commented out this code also. Not sure if you were suggesting that I delete this exercise file also?